spec/features/staff/users_can_create_a_budget_spec.rb

```diff
@@ -177,6 +179,39 @@
 
         expect(page).to have_content(t("action.budget.create.success"))
       end
+
+      scenario "for a direct budget it sets the providing org to the service owner (BEIS)" do
+        _report = create(:report, state: :active, organisation: user.organisation, fund: fund_activity)
+
+        visit activities_path
+        click_on(project_activity.title)
+        click_on(t("page_content.budgets.button.create"))
+
+        choose(user.organisation.name)
+        select "#{Date.current.year}-#{Date.current.next_year.year}", from: "budget[financial_year]"
+        fill_in "budget[value]", with: "1000.00"
+        click_button t("default.button.submit")
+
+        expect(page).to have_content(t("action.budget.create.success"))
+        expect(project_activity.budgets.last.providing_organisation.service_owner).to eql(true)
```

Comment:
Would something like this be more readable:

`expect(project_activity.budgets.last.providing_organisation).to eql(beis)`

NB: `beis` isn't currently being set.